projects/wave-core-new/src/lib/project/project.service.ts

```diff
@@ -1277,6 +1277,16 @@ export class ProjectService {
             );
     }
 
+    private addTimeToProperties(x: any): any {
+        x['features'].forEach((element: any) => {
+            let start: string = element['when']['start'];
+            let end: string = element['when']['end'];
+            element['properties']['start'] = start;
+            element['properties']['end'] = end;
```

Comment:
I guess you should use keys that are less likely to have name conflicts since you just override stuff.

Reply:
I renamed them to table__start and table__end (two underscores each) respectively, that should be OK.